snap/info.go

```diff
@@ -1005,28 +1005,28 @@ func DropNick(nick string) string {
 	return nick
 }
 
-// StoreName splits the instance name and returns the store name of the snap.
-func StoreName(instanceName string) string {
-	storeName, _ := SplitInstanceName(instanceName)
-	return storeName
+// SnapName splits the instance name and returns the name of the snap.
+func SnapName(instanceName string) string {
```

Comment:
Per comment in prior PR, that seems better as InstanceSnap.. otherwise it lacks context (snap name what?).